pkg/apis/cni/v1beta1/cni.proto

```diff
@@ -18,7 +18,7 @@ import "google/protobuf/any.proto";
 
 package antrea_io.antrea.pkg.apis.cni.v1beta1;
 
-option go_package = "v1beta1";
+option go_package = "pkg/apis/cni/v1beta1";
```

Comment:
I agree this go package format makes more sense. 
I wanted to move to this as well but skipped my mind as tests were passing with generated go code.
I will move to this.

Reply:
Tried this but when generating go code through Antrea makefile, I see that code is generated in a new folder "antrea.io/antrea" in the repo root. Instead of making changes to the makefile, as per offline discussion, we decided to keep the go package path as it is.